java/src/jmri/jmrit/display/Editor.java

```diff
@@ -3056,6 +3056,13 @@ public void dispose() {
         }
         InstanceManager.getDefault(EditorManager.class).remove(this);
         setVisible(false);
+        for (Positionable _content : _contents) {
+            if (_content.getLogixNG() != null) {
+                LogixNG logixNG = _content.getLogixNG();
+                deleteLogixNG_Internal(logixNG);
+                logixNG.dispose();
```

Comment:
I don't think the LogixNG should be disposed after it has been deleted. So I think this line should be removed:

   `logixNG.dispose();`